sbt-scalaxb/src/main/scala/sbtscalaxb/ScalaxbPlugin.scala

```diff
@@ -126,7 +126,9 @@ object ScalaxbPlugin extends sbt.AutoPlugin {
         (if (scalaxbVararg.value && !scalaxbGenerateMutable.value) Vector(VarArg) else Vector()) ++
         (if (scalaxbGenerateMutable.value) Vector(GenerateMutable) else Vector()) ++
         (if (scalaxbGenerateVisitor.value) Vector(GenerateVisitor) else Vector()) ++
-        (if (scalaxbAutoPackages.value) Vector(AutoPackages) else Vector())
+        (if (scalaxbAutoPackages.value) Vector(AutoPackages) else Vector()) ++
+        (if (scalaxbCapitalizeWords.value) Vector(CapitalizeWords) else Vector()) ++
```

Comment:
I think this setting needs to be initialized somewhere.

```scala
scalaxbCapitalizeWords := false
```

Reply:
I tried to do this around line 86 but then I found that it overrides the value I set in the sbt-test project. Did I miss something?

Reply:
Then the setting would set by the user as:

```scala
scalaxbCapitalizeWords in (Compile, scalaxb) := true,
```

Reply:
Done.